src/includes/environment.js

```diff
@@ -461,61 +461,68 @@ pub.layer = function(layer) {
 
 
 /**
- *  Returns the Group instance and sets it if argument Group is given.
+ *  Groups items to a new group. Returns the resulting group instance. If a string is given as the only
+ *  argument, the group by the given name will be returned.
  *
  *  @cat Document
  *  @subCat Page
- *  @method Group
- *  @param {Array} [pItem] The PageItems array (must be at least 2) or name of Group name instance.
- *  @param {String} name The name of the Group, only when creating a Group from Page Item(s).
- *  @return {Group} The current Group instance.
+ *  @method group
+ *  @param {Array} pItems An array of page items (must contain at least two items) or name of group instance.
+ *  @param {String} [name] The name of the group, only when creating a group from page items.
+ *  @return {Group} The group instance.
  */
-pub.group = function (pItem, name) {
-  checkNull(pItem);
-  var group = null;
-  if(pItem instanceof Array) {
-    if(pItem.length < 2) {
-      error("There must be at least two PageItems passed to b.group().");
+pub.group = function (pItems, name) {
+  checkNull(pItems);
+  var group;
```

Comment:
Why not setting it `null`? I often do that to not have an undefined variable

Reply:
Hm, I only set variables to `null`, if I need them to be `null` for some reason. Here I don't need them to be anything for the moment, I just need to initialize them, so I do it in the shortest, most concise way possible.

Assigning `null` would feel a bit like.

```js
var group = "temp";
```

As in "I just assign some value, although I don't need it anyways".

Also, without assigning `null`, I find it is much cleaner and easier to initialize several variables:

```js
var foo, bar, baz;
```

I just checked the basil.js source and there are many places, where `var foo;` is used to initialize.

What do you think?

Reply:
I like my variables to be `null`. :-) So I can always check if they are null or something else. I found `undefined` to be sometimes not reliable for checking. Don't bother. Just a question.  

It could also be:  

    var x = y = z = null;

Reply:
Assigning null is for me different then giving a string a content. On a string I would go for 

    var thing = ''; // or null

Reply:
```js
var x = y = z = null;
```
Yes, I don't find this as clean though. No matter, if `undefined` is reliable for checking or not, in this case I know that I *don't* need to check it, that's why I think, I don't need to assign anything, not even `null`. And you're right, the string comparison does not really work.

I guess it just comes down to style. Googling also revealed that some do it like this, others like that.

> Don't bother. Just a question.

Cool, so I will just keep it like this now, okay?

Reply:
> Cool, so I will just keep it like this now, okay?
Yes. If not we should change things like this throughout basil.

Reply:
Ok, good to merge then, right?

Reply:
For me. Yes. 👍